Migration of account generation page to support pages

filippoweb3 opened the pull request:
Left some tiny intro and a list of ways to create accounts with links to support pages

Commit:
Migrated everything

Left some tiny intro and a list of ways to create accounts with links to support pages

profradha commented on docs/learn/learn-account-generation.md:
```diff
@@ -8,415 +8,21 @@ slug: ../learn-account-generation
 
 An address is the public part of a Polkadot account. The private part is the key used to access this
-address. The public and private parts together make up a Polkadot account.
+address. The public and private parts together make up a Polkadot account. You can think about your public address like your mailbox and your private key like your key to open your mailbox. Everybody can send you information to your public address, but only you can access the box and read such information. In other words, everybody can know your public address but you should keep your private key ...
```
I'm thinking if there are better words to to explain this than just saying public and private part.

Rephrased it a little bit. Feel free to edit it further

```suggestion
address. The public and private parts together make up a Polkadot account. You can think of the public address  of your account like your mailbox and the private key like the key to open that mailbox. Anybody can send mail to your mailbox, but only you can access them as only you have access to its key. In the context of Polkadot accounts,  anybody can send tokens to your public address but only you can transact with them using your private key. That is why you should keep your private key secret.
```

DrW3RK committed:
Update the doc

DrW3RK reviewed (changes requested):
I have some feedback that can help with future PRs as well.

1. It is important to ensure the content being deleted on the Wiki is present in the support article being referenced. 
2. It is okay to have redundant information on the Wiki page as well as the Support page. The ultimate goal is to have a coherent reading experience for the user

You have put a lot of effort to create the password reset section on the Wiki, but deleted it without ensuring it is transmitted to the support page. I tried to retain the backbone of this Wiki document with relevant references to the support articles. Let me know what you think from the perspective of a new reader landing on this Wiki page.

filippoweb3 committed:
Update docs/learn/learn-account-generation.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 commented:
I overlooked it and did not notice I deleted all that part I worked on! My apologies, I agree the page must be like you mentioned. I am going through it and check if it need further additions / rearrangements.

Commit:
Removed pics old extension

Removed picture showing switching address formats with old extension, addition to support article. Bug fix of some internal links. Minor edits.

Commit:
Merge branch 'account-generation-migration' of https://github.com/w3f/polkadot-wiki into account-generation-migration

profradha commented on docs/learn/learn-account-generation.md:
```diff
@@ -8,16 +8,17 @@ slug: ../learn-account-generation
 There are several ways to generate a Polkadot account:
 
-- [Polkadot{.js} Browser Extension](#polkadotjs-browser-extension) **RECOMMENDED FOR MOST USERS**
+- [Polkadot{.js} Browser Extension](#polkadot-js-apps) **RECOMMENDED FOR MOST USERS**
```
Have you mistakenly pointed this ref link to point to Polkadot JS Apps?

```suggestion
- [Polkadot-JS Browser Extension](#polkadot-js-browser-extension) **RECOMMENDED FOR MOST USERS**
```

profradha commented on docs/learn/learn-account-generation.md:
```diff
@@ -39,98 +40,25 @@ security trade-offs. Please review this page carefully before making your accoun
-
-The Polkadot{.js} Extension provides a reasonable balance of security and usability. It provides a
+The Polkadot JS Extension provides a reasonable balance of security and usability. It provides a
 separate local mechanism to generate your address and interact with Polkadot.
```
Staying consistent with Glossary https://support.polkadot.network/support/solutions/articles/65000110873-glossary and at the same time, preserving the link to this section referenced on third party websites
```suggestion
The Polkadot-JS Browser Extension (simply referred to as Polkadot Extension) provides a reasonable balance of security and usability. It provides a
separate local mechanism to generate your address and interact with Polkadot.
```

profradha commented on docs/learn/learn-account-generation.md:
```diff
@@ -39,98 +40,25 @@ security trade-offs. Please review this page carefully before making your accoun
 separate local mechanism to generate your address and interact with Polkadot.
 
-This method involves installing the Polkadot{.js} plugin and using it as a “virtual vault," separate
+This method involves installing the Polkadot JS extension and using it as a “virtual vault," separate
```
```suggestion
This method involves installing the Polkadot Extension and using it as a “virtual vault," separate
```

profradha commented on docs/learn/learn-account-generation.md:
```diff
@@ -39,98 +40,25 @@ security trade-offs. Please review this page carefully before making your accoun
-![plugin-02](../assets/accounts/polkadot_plugin_js_02.jpg)
-
-### Create Account
+:::info For the tutorial on how to create an account using Polkadot JS browser extension, visit [this support article](https://support.polkadot.network/support/solutions/articles/65000098878-how-to-create-a-dot-account). 
```
```suggestion
:::info For the tutorial on how to create an account using Polkadot Extension, visit [this support article](https://support.polkadot.network/support/solutions/articles/65000098878-how-to-create-a-dot-account). 
```

profradha commented on docs/learn/learn-account-generation.md:
```diff
@@ -172,36 +100,24 @@ networks without having to set different passwords to access such accounts.
-Click on "Options" at the top of the plugin window, and under "Display address format for" select
-"Polkadot Relay Chain".
+:::info For privacy reasons, we recommend creating a new address for each chain you're using.
 
```
Reasoning behind the recommendation 

```suggestion

Your address' format is only visual - the data used to derive this representation of your address
are the same, so you can use the same address on multiple chains. However, for privacy reasons, we
recommend creating a new address for each chain you're using.
```

DrW3RK reviewed (approved):
Looks good. Made a few suggestions. Please address them and then we can merge this

filippoweb3 committed:
Update docs/learn/learn-account-generation.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-account-generation.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-account-generation.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-account-generation.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-account-generation.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>
